src/app/home/avatar-generator/avatar-generator.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AvatarGeneratorComponent } from './avatar-generator.component';
+
+describe('AvatarGeneratorComponent', () => {
+  let component: AvatarGeneratorComponent;
+  let fixture: ComponentFixture<AvatarGeneratorComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ AvatarGeneratorComponent ],
```

Comment:
This test suite is failing because `SanitizeHtmlPipe` cannot be found, please add it to the `declarations` array of the test module.